src/components/include/cbuf.h

```diff
@@ -265,14 +265,12 @@ cbuf2buf(cbuf_t cb, int len)
 	do {
 		cm = cbuf_vect_lookup_addr(id);
 		if (unlikely(!cm || CBUF_PTR(cm) == 0)) {
-			printc("no cm for %d!\n", id);
+			printc("cbuf::no cm for %d!\n", id);
```

Comment:
I don't know why this print statement is in the mainline.  @ryuxin must have included it.  I'm guessing we don't want this, and the print below?